### DISCUSSION TITLE
How to get eslint-plugin-react-compiler with biome?

### DISCUSSION THREAD

**Discussion by MariuzM** [biomejs/biome#2999] [General]
With Eslint and React 19 + React Compiler there is eslint `eslint-plugin-react-compiler` how can i use this with biome?

**Comment by arendjr** [comment]
This is being discussed here: https://github.com/biomejs/biome/discussions/3#discussioncomment-9470057

Thanks!

**Reply by mrackwitz** [reply]
FYI for everyone finding this, the discussion has moved to: https://github.com/biomejs/biome/discussions/5290
